test/remote/gateways/remote_mondido_test.rb

```diff
@@ -0,0 +1,1487 @@
+require 'test_helper'
+
+class RemoteMondidoTest < Test::Unit::TestCase
```

Comment:
So that's a 'no' for ones we can make public? I'll take them privately, but would love to have a set we can commit here and use in CI